diff --git a/node_modules/enhanced-resolve/lib/ResolverFactory.js b/node_modules/enhanced-resolve/lib/ResolverFactory.js
index 508e2a8..e8b9f93 100644
--- a/node_modules/enhanced-resolve/lib/ResolverFactory.js
+++ b/node_modules/enhanced-resolve/lib/ResolverFactory.js
@@ -72,7 +72,8 @@ exports.createResolver = function(options) {
 
 	// Resolve symlinks to their symlinked location
 	const symlinks =
-		typeof options.symlinks !== "undefined" ? options.symlinks : true;
+       // FIXME: we should set resolve.symlinks=false where Angular CLI creates the webpack config
+		typeof options.symlinks !== "undefined" ? options.symlinks : false;
 
 	// Resolve to a context instead of a file
 	const resolveToContext = options.resolveToContext || false;
